packages/demo/examples/01-xy-chart/index.jsx

```diff
@@ -15,6 +15,8 @@ import {
   LineSeries,
   PointSeries,
   StackedBarSeries,
+  BoxPlotSeries,
```

Comment:
since this file is getting so big I think it'd be good to start moving the examples into their own files. I did this with `<StackedAreaSeries />`, could we do that here?

Reply:
sounds good! I moved them out as a separate file